MacOS/Python/Install.sh

```diff
@@ -62,13 +62,32 @@ _py_version=$PYTHON_VERSION_PS
 # Install miniconda
 # Check if miniconda is installed
 
-echo "Installing Miniconda..."
+echo "Checking for Miniconda or Anaconda installation..."
 if conda --version > /dev/null; then
-  echo "Miniconda or anaconda is already installed"
-else
-  brew install --cask miniconda
-  [ $? -ne 0 ] && exit_message
+    echo "Miniconda or Anaconda is already installed."
+    # print conda version 
+    echo "Conda version: $(conda --version)"
+    clear -x 
+    echo "To proceed with the installation, the existing Conda installation must be removed."
+    read -p "Do you want to uninstall it? (Type 'yes' to confirm): " confirm
+    if [ "$confirm" = "yes" ]; then
```

Comment:
Perhaps we should do things a bit more succintly.

```shell
conda info --base
```
will give you the path of where it is installed. And if it is a brew installed one, then that is ok.
This would also allow a more direct installation of the script it self.

Reply:
resolved in https://github.com/dtudk/pythonsupport-scripts/pull/51/commits/da5e210fa3d52a6f5550be6e67eae02521d9f0ab (this adresses a problem in the initial fix)